amity/Model/person.py

```diff
@@ -0,0 +1,17 @@
+class Person (object):
+
+    def __init__(self, person_name, person_type, allocate_living_space="N"):
+        self.person_name = person_name
+        self.person_type = person_type
+
+
+class Fellow(Person):
+    def __init__(self, person_name):
+        #overrides the person_type
```

Comment:
Its kinda confusing, I though best practices included both docstrings and comments